packages/cozy-konnector-libs/docs/api.md

```diff
@@ -451,6 +454,36 @@ You need the full permission on `io.cozy.files` in your manifest to use this fun
 ```javascript
 await saveFiles([{fileurl: 'https://...', filename: 'bill1.pdf'}], fields)
 ```
+<a name="module_saveIdentity"></a>
+
+## saveIdentity
+Helper to set or merge identities
```

Comment:
A link to https://github.com/cozy/cozy-doctypes/blob/master/docs/io.cozy.identities.md would be better for documentation